libs/openFrameworks/3d/ofEasyCam.h

```diff
@@ -163,52 +160,70 @@ class ofEasyCam : public ofCamera {
 	/// Returns the area bounds used for mouse control.
 	ofRectangle getControlArea() const;
 	
+	
+	enum easyCamTransformType{
+		EASYCAM_TRANSFORM_NONE,
```

Comment:
Likewise here, can we change these to `TRANSFORM_TYPE_NONE`, `TRANFORM_TYPE_ROTATE`, etc?